File: JavaScript/JavaScript-Fundamentals/Operators and Expressions/Operators and Expressions/OddOrEven.js
function isOdd(num) {
    return (num % 2 === 0) ? true : false;
}
console.log('***********************************');
console.log('Problem 1: Odd or Even:');
console.log('***********************************');

for (var i = -5; i < 5; i+=1) {
    console.log("Is " + i + " odd: " + isOdd(i));
}

File: CSharp1/OperatorsAndExpressions/OperatorsAndExpressions/OddOrEven.cs
// Problem 1: Write an expression that checks if given integer is odd or even.

using System;

class OddOrEven
{
    static void Main()
    {
        Console.Write("Write an integer:");
        int number = int.Parse(Console.ReadLine());

        if (number % 2 == 0)
        {
            Console.WriteLine("The number is even!");
        }
        else
        {
            Console.WriteLine("The number is odd!");
        }
    }
}

